src/utils/createMeshFromObject.test.ts

```diff
@@ -0,0 +1,121 @@
+import * as THREE from "three";
+import { mergeVertices } from "three/examples/jsm/utils/BufferGeometryUtils";
+import { beforeEach, describe, expect, it, vi } from "vitest";
+import type { VertexUsageInfo } from "@/types";
+import { createMeshFromObject } from "./createMeshFromObject";
+
+vi.mock("three/examples/jsm/utils/BufferGeometryUtils", () => ({
```

Comment:
Tests are fine; they are a great starting point for reviewing the code. Thank you